plugins/package-managers/spm/src/main/kotlin/SpmModel.kt

```diff
@@ -34,10 +34,10 @@ import org.ossreviewtoolkit.utils.ort.normalizeVcsUrl
 val json = Json { ignoreUnknownKeys = true }
 
 /**
- * The output of the `spm dependencies` command.
+ * The data model for the output of the command `swift package show-dependencies --format json`.
```

Comment:
Out of curiosity: Was `spm dependencies` the correct command at some point in time, or was it always wrongly documented?

Reply:
When the initial implementation was merged, the command was named `swift`, not `spm`. Do, I believe it's been wrongly documented from the beginning.